File: app/src/main/java/com/example/getfood/ui/splash/SplashMvpView.java
package com.example.getfood.ui.splash;

import com.example.getfood.ui.base.BaseView;

public interface SplashMvpView extends BaseView {
    void userIsSignedIn();

    void userIsNotSignedIn();

    void updateNotRequired();

    void updateRequired();
}

File: app/src/main/java/com/example/getfood/ui/splash/SplashMvpView.kt
package com.example.getfood.ui.splash

import android.content.Context
import com.example.getfood.ui.base.BaseView

interface SplashMvpView : BaseView {
    val context: Context
    fun userIsSignedIn()
    fun userIsNotSignedIn()
    fun updateNotRequired()
    fun updateRequired()
}